styles 2.css

```diff
@@ -0,0 +1,76 @@
+body {
+  font-family: Arial, sans-serif;
+  background-color: #f4f4f4;
+  padding: 20px;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 5px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+h1 {
+  text-align: center;
+  margin-bottom: 20px;
+}
+
+#note-form {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+input, textarea {
+  padding: 10px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+}
+
+button {
+  padding: 10px;
+  background-color: #007bff;
+  color: #fff;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+button:hover {
+  background-color: #0056b3;
+}
+
+#notes-list 
+  margin-top: 20px;
+}
+
+.note {
+  background-color: #f8f9fa;
+  border: 1px solid #dee2e6;
+  padding: 10px;
+  border-radius: 5px;
+  margin-bottom: 10px
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.note-title, .note-content {
+  margin: 0;
+}
+
+.delete-btn {
+  background-color: #dc3545;
+  padding: 5px;
+  border: none;
+  border-radius: 5px;
+  color: #fff;
+  cursor: pointer;
+
+
+.delete-btn:hover {
```

Comment:
The '.delete-btn' CSS rule set is not properly closed with a curly brace. Please add a closing curly brace.